### PR TITLE
Duplicate function name on line 303 and 182

### PR DESCRIPTION
**Pull request by deenaik**
'test_count_vector_featurizer' on line 303 renamed to 'test_count_vector_featurizer_ab_words'

**Proposed changes**:
-  Duplicate function name on line 303 and 182
'test_count_vector_featurizer' on line 303 renamed to 'test_count_vector_featurizer_ab_words'

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
thanks for submitting this PR, we'll give it a review as soon as possible

File: tests/nlu/base/test_featurizers.py [hunk 1]
```diff
@@ -300,7 +300,7 @@ def test_count_vector_featurizer_using_tokens(tokens, expected):
         ("abc", [1, 1, 1, 1, 1]),
     ],
 )
-def test_count_vector_featurizer(sentence, expected):
+def test_count_vector_featurizer_ab_words(sentence, expected):
```
Comment (Ghostvv) [inline tests/nlu/base/test_featurizers.py]:
```suggestion
def test_count_vector_featurizer_char(sentence, expected):
```

**Comment by deenaik** [comment]
updated thanks @Ghostvv

**Comment by deenaik** [comment]
Hey @Ghostvv can you merge this pull request now.

**Pull request merged**